File: sprint2/By lessons/lesson7code1/lesson7code1.cs
using System;
using System.Collections.Generic;

class Stack {
    private List<string> items;

    public Stack() {
        items = new List<string>();
    }

    public void Push(string item) {
        items.Add(item);
    }

    public string Pop() {
        string lastItem = items[items.Count - 1];
        items.RemoveAt(items.Count - 1);
        return lastItem;
    }

    public string Peek() {
        return items[items.Count - 1];
    }

    public int Size() {
        return items.Count;
    }
}

class Program {
    static void Main(string[] args) {
        Stack stack = new Stack();
        stack.Push("apple");
        stack.Push("banana");
        stack.Push("orange");
        stack.Pop();
    }
}

File: sprint2/By lessons/lesson7code1/lesson7code1.py
class Stack:
     def __init__(self):
         self.items = []

     def push(self, item):
         self.items.append(item)

     def pop(self):
         return self.items.pop()

     def peek(self):
         return self.items[-1]

     def size(self):
         return len(self.items)
stack = Stack()
stack.push('apple')
stack.push('banana')
stack.push('orange')
stack.pop()
